www/js/app.js

```diff
@@ -33,8 +37,17 @@ angular.module('building-blocks', ['ionic', 'building-blocks.controllers', 'buil
         controller: 'HomeController'
       }
     }
-  });
+  })
 
+  .state('tab.message', {
+    url: '/message',
```

Comment:
why aren't you using `help_request`, IMO it's good to keep some consistency on how you name your resources and associated routes. It helps when reading the code, I can immediately tell what's what.